doc/source/quickstart.rst

```diff
@@ -93,11 +93,34 @@ named ``tools``, and you will need to run extra commands to install
 the SDK packages needed. 
 
 For Android NDK, note that modern releases will only work on a 64-bit
-operating system. If you are using a 32-bit distribution (or hardware),
-the latest useable NDK version is r10e, which can be downloaded here:
+operating system. The minimal, and recommended, NDK version to use is r17c:
+
+ - `Linux (64-bits) <https://dl.google.com/android/repository/android-ndk-r17c-linux-x86_64.zip>`_
```

Comment:
@inclement, I put the direct link because the main page for downloads does not contain the ndk version r17c, plus the ndk r18 seems to have issues documented by @tito in #1525 which right now maybe solved with the latests changes I applied en here...